powerauth-push-server/src/main/java/io/getlime/push/service/InboxService.java

```diff
@@ -124,13 +117,18 @@ public GetInboxMessageDetailResponse readMessage(String userId, String appId, St
 
     @Transactional
     public void readAllMessages(String userId, String appId) throws AppNotFoundException {
-        checkAppId(appId);
-        int countRead = inboxRepository.markAllAsRead(userId, appId);
+        final AppCredentialsEntity appCredentialsEntity = checkAppId(appId);
+        int countRead = inboxRepository.markAllAsRead(userId, appCredentialsEntity);
         logger.info("Marked all inbox messages as read for user: {}, count: {}", userId, countRead);
     }
 
-    private void checkAppId(String appId) throws AppNotFoundException {
-        appCredentialsRepository.findFirstByAppId(appId).orElseThrow(
+    private AppCredentialsEntity checkAppId(String appId) throws AppNotFoundException {
```

Comment:
I would not call this method `checkAppId` anymore.

Reply:
Agreed.